src/Service/Extensions/ConversionFunctions.cs

```diff
@@ -0,0 +1,6 @@
+namespace Passwordless.Service.Extensions;
+
+public static class ConversionFunctions
+{
+    public static TimeSpan? ToTimeSpanFromSeconds(this int seconds) => TimeSpan.FromSeconds(seconds);
```

Comment:
One last thing (sorry I must've missed this in my own code snippet)

```suggestion
    public static TimeSpan ToTimeSpanFromSeconds(this int seconds) => TimeSpan.FromSeconds(seconds);
```